test/library/input.spec.js

```diff
@@ -313,5 +313,13 @@ describe('NumberFormat as input', () => {
         shallow(<NumberFormat format="#### #### ####" mask={['D', 'D', 'M', '1', '2', 'Y', 'Y', 'Y']}/>)
       }).toThrow()
     })
-  })
+  });
+
+  it('should allow negationFormat to be parentheses', () => {
```

Comment:
The spec statement is not very clear for the thing you are testing. You are only checking for empty/NAN values.